reference/managed-delivery/resource-status/index.md

```diff
@@ -0,0 +1,160 @@
+```
+Example response:
+```json
+{ "status": "HAPPY" }
+```
+
+## Status Reference
+
+### Happy
+{%
+  include
+  figure
+  image_path="./resource-status-happy.png"
+  caption="Example of a resource with the Happy status"
+%}
+
+When a resource has a Happy status, all is well and the actual state of the resource matches its declarative configuration.
+
+If Spinnaker detects a drift from the declarative configuration in the future, the resource will change status to [Diff](/reference/managed-delivery/resource-status/#diff). Once Spinnaker starts taking automatic action to correct the drift, the status will change to [Actuating](/reference/managed-delivery/resource-status/#actuating).
+
+While Spinnaker is managing a resource, manual changes made via the UI or the API will be automatically reversed ("stomped") in favor of the declarative configuration.
+
+### Created
+{%
+  include
+  figure
+  image_path="./resource-status-created.png"
+    caption="Example of a resource with the Created status"
+%}
+
+When a resource has a Created status, it means Spinnaker was just told to start management and hasn't taken any action yet.
+
+If the actual state of the resource matches its declarative configuration, the resource will transition to the [Happy](/reference/managed-delivery/resource-status/#happy) status the next time Spinnaker checks it.
+
+If Spinnaker detects a drift from the declarative configuration in the future, the resource will change status to [Diff](/reference/managed-delivery/resource-status/#diff). Once Spinnaker starts taking automatic action to correct the drift, the status will change to [Actuating](/reference/managed-delivery/resource-status/#actuating).
+
+While Spinnaker is managing a resource, manual changes made via the UI or the API will be automatically reversed ("stomped") in favor of the declarative configuration.
+
+### Diff
+{%
+  include
+  figure
+  image_path="./resource-status-diff.png"
+  caption="Example of a resource with the Diff status"
+%}
+
+When a resource has a Diff status, it means Spinnaker detected a drift from the declarative configuration but hasn't taken any action yet.
+
+Once Spinnaker starts taking automatic action to correct the drift, the status will change to [Actuating](/reference/managed-delivery/resource-status/#actuating).
+
+While Spinnaker is managing a resource, manual changes made via the UI or the API will be automatically reversed ("stomped") in favor of the declarative configuration.
+
+### Actuating
+{%
+  include
+  figure
+  image_path="./resource-status-actuating.png"
+  caption="Example of a resource with the Actuating status"
+%}
+
+When a resource has an Actuating status, it means Spinnaker detected a drift from the declarative configuration and automatic action is in progress to resolve it. You can go to the Tasks view in your app to see the actions Spinnaker is taking on your behalf, and whether they succeeded or failed.
```

Comment:
```suggestion
When a resource has an Actuating status, it means Spinnaker detected a drift from the declarative configuration and an automatic action is in progress to resolve it. You can go to the Tasks view in your app to see the actions Spinnaker is taking on your behalf, and whether they succeeded or failed.
```

Reply:
🤷‍♂ I've been thinking of it as "N" actions instead of one, because I guess theoretically we might choose to dispatch multiple tasks at a time? Either way I'm fine with this since ideally the history view will tell you about the diff -> task(s) mapping.

Reply:
Specifically we do launch one separate task per region for most resource types